cmd/brew-services.rb

```diff
@@ -155,9 +160,18 @@ def list
       puts format("#{Tty.white}%-#{longest_name}.#{longest_name}s %-7.7s %-#{longest_user}.#{longest_user}s %s#{Tty.reset}",
                   "Name", "Status", "User", "Plist")
       formulae.each do |formula|
+        status = case formula[:status]
+                 when :started
+                   "#{Tty.green}started#{Tty.reset}"
+                 when :stopped
+                   "stopped"
+                 when :error
+                   "#{Tty.red}error  #{Tty.reset}"
```

Comment:
Yes, exactly! It felt like a proper alignment of the output is a bit out of scope for this PR :)